packages/cli/src/init.ts

```diff
@@ -246,7 +249,12 @@ export function generateScript(project: { name: string }): string {
 export function generateTSConfig(): string {
     return detent(`
         {
-            "extends": "@gitbook/tsconfig/integration.json"
+            "extends": "@gitbook/tsconfig/integration.json",
+            "compilerOptions": {
+                "lib": ["ES2015", "DOM"],
```

Comment:
I think you can go higher `ESNext` or something. Let's also add `strict: true` as that has been the source of many unwanted errors in the past